### PR TITLE
[#104] weather widget

### PR DESCRIPTION
**Pull request by mpugach**
- [x] User can set postal code
- [x] User can change postal code
- [x] Postal code is mapped to location
- [x] Weather is retrieved by location and checkin date
- [x] Weather is cached into DB for given date and postal code
- [x] Weather summary is shown in weather widget
- [x] Add index to Weather table
- [x] Add tests
- [x] Complete widget markup
- [x] Implement charts markup
- [x] Get new weather on new checkin if postal code was set on previous checkin
- [x] add status icon

### CODE REVIEW

File: frontend/app/mixins/graphable.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
```
Comment (safeforge) [inline frontend/app/mixins/graphable.js]: @mpugach what do you think to move this  in frontend/app/components/chart ?

Comment (mpugach) [reply frontend/app/mixins/graphable.js]: do you mean to create some component only to extend it with `g-trackable` and `g-weather`?

Comment (safeforge) [reply frontend/app/mixins/graphable.js]: nope, just to move mixin into compoments/char directory such as `draggable.js`

Comment (mpugach) [reply frontend/app/mixins/graphable.js]: @safeforge what do you think about `frontend/app/components/chart/mixins` subdir?

Comment (safeforge) [reply frontend/app/mixins/graphable.js]: I'm working on new health-chart components and I'd like to have all stuff into `chart` directory.

**Pull request merged**